.gitignore

```diff
@@ -16,3 +16,6 @@ api-docs/
 _site/
 
 .DS_Store
+
+# Ignore env files
+.env
```

Comment:
🧗‍♀️ Can we remove the checked in `.env` file? Having `.env` in ignore while shipping with an `.env` file could be a little confusing. 🙈

Reply:
I'll drop it from the ignore file. I added it so I wouldn't accidentally add the GA credentials. I want to ship with the .env file so folks can use my Contentful credentials and get a functional build straight away.